apps/test/unit/templates/teacherDashboard/teacherSectionsReduxTest.js

```diff
@@ -112,8 +118,10 @@ const sections = [
     pairing_allowed: false,
     sharing_disabled: false,
     course_offering_id: 3,
+    courseOfferingId: 3,
     course_version_id: 5,
     unit_id: 7,
+    unitId: 7,
```

Comment:
Can you remind me what we need both versions of all these variables for (like unitId and unit_id)?

Reply:
good catch! I thought we needed this but I'm pretty sure we don't. I'll remove